comp/api/api/apiimpl/grpc.go

```diff
@@ -113,14 +114,13 @@ func (s *serverSecure) DogstatsdSetTaggerState(_ context.Context, req *pb.Tagger
 	// Reset and return if no state pushed
 	if req == nil || req.State == nil {
 		log.Debugf("API: empty request or state")
-		s.taggerComp.ResetCaptureTagger()
```

Comment:
I think this was the only call to `ResetCaptureTagger` so it can already be removed from the interface and the different implementations.
Same for `ReplayTagger` and `SetNewCaptureTagger` called below.

Reply:
I’m okay with whatever you think will be easier.
I thought about removing everything that’s not used anymore to keep the main branch clean.

Reply:
Let's do it now, I will also address @pgimalac comment at the same time 👍